src/SelfAssessment/AssessmentStart.spec.tsx

```diff
@@ -1,9 +1,15 @@
-import { render, fireEvent } from "@testing-library/react-native"
 import React from "react"
+
+import { render, fireEvent } from "@testing-library/react-native"
 import { I18nextProvider } from "react-i18next"
+import { useNavigation } from "@react-navigation/native"
 
 import i18n from "../locales/languages"
-import { AssessmentStart } from "./AssessmentStart"
+import AssessmentStart from "./AssessmentStart"
+
+const navigationSpy = jest.fn()
+jest.mock("@react-navigation/native")
```

Comment:
You will most likely need to leave this line up here because even if we don't use the exact value for the mock in each test, the hook is bein called.

Reply:
Ah I see thank you!